guides/common/modules/proc_loading-the-default-openscap-contents.adoc

```diff
@@ -0,0 +1,37 @@
+[id="Loading_the_Default_OpenSCAP_Contents_{context}"]
+= Loading the Default OpenSCAP Contents
+
+On {ProjectServer}, load the default OpenSCAP contents using one of the following methods in the CLI.
+
+This procedure ensures that data streams available in `/usr/share/xml/scap/ssg/content/` are loaded and assigned to all organizations and locations.
+
+.Prerequisites
+* Your user account has the `create_scap_contents` permission.
+* Ensure that the SCAP Security Guide package is installed on {ProjectServer}:
```

Comment:
Does this depend on the OS of Foreman Server or the OS of managed hosts? Aka. I can run Foreman on EL and manage Ubuntu/SLES/RHEL; what content would I need to install?

Reply:
Actually, I mention it in the second sentence in the following section, _Uploading Additional SCAP Content_.
Not sure if it's the right place, but the user has to handle it as additional content....